src/include/homestore/replication/repl_decls.h

```diff
@@ -78,6 +78,13 @@ struct peer_info {
     uint64_t last_succ_resp_us_;
 };
 
+struct replica_member_info {
+    static constexpr uint64_t max_name_len = 32;
```

Comment:
I am concerning the 32 is enough.  UUID is 128bit however string representation of UUID is 36byte due to 4 "-" are added.   And plus the "\n" it should at least 37.

As PG member change is rare,  maybe we can trade space with convenient make it 64bytes or even longer?

Reply:
sorry it is 16bytes....

Reply:
update:  CM use the name of SM when SM register to CM.  SM fill that with POD NAME atm.
https://github.corp.ebay.com/SDS/storage_mgr/blob/main/src/lib/sm_lib.cpp#L345-L347

Which makes the fix_len more risky , we may want to 
a). use json serialization/ deserialization similar to what we do for create_pg. 
b). make this limit larger and enforcing checks

Reply:
Made it 128 bytes which should be good enough